style.css

```diff
@@ -623,3 +622,62 @@ button:active {
     align-items: center;
   }
 }
+
+.overlay-bg {
+  /* Height & width depends on how you want to reveal the overlay (see JS below) */
+  height: 100%;
+  width: 0;
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  background-color: #505ccacb;/* Black fallback color */
+  overflow-x: hidden; /* Disable horizontal scroll */
+  transition: 0.5s; /* 0.5 second transition effect to slide in or slide down the overlay (height or width, depending on reveal) */
+}
+
+.overlay-bg .closebtn {
+  padding: 0;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.299);
+  float: right;
+  margin: 24px;
+  transition: 0.3s; /* Transition effects on hover (color) */
+  border: 2px solid #67798e;
```

Comment:
- [ ] Your design is showing a border in this element which is not part of the Figma design. I think it could be a good idea to remove the border for this element in order to match the template :wink:
![image](https://user-images.githubusercontent.com/81522435/145887573-d114a31e-5d4a-441a-993e-acf2a6e956f8.png)

Reply:
:+1: Addressed :heavy_check_mark: